src/main/scala/com/fulcrumgenomics/sv/tools/AggregateSvPileup.scala

```diff
@@ -159,49 +158,34 @@ object AggregateSvPileup {
    * Converts a set of pairwise neighbor relationships to a set of clusters, where each cluster consists of breakpoints
    * that can be joined into a connected path via pairwise neighbor relationships.
    * @param pileupToNeighbors Map of pileup to its set of neighbors
-   * @param existingClusters An existing set of clusters to add to
    * @return The set of pileup clusters
    */
-  @tailrec
-  def toClusters(pileupToNeighbors: Map[BreakpointPileup, PileupGroup],
-                 existingClusters: Seq[PileupGroup] = Seq()): Seq[PileupGroup] = {
-    pileupToNeighbors.headOption match {
-      case None              => existingClusters
-      case Some((pileup, _)) =>
-        val (pileupGroup, remaining) = extractClusterFor(pileup, pileupToNeighbors)
-        toClusters(remaining, existingClusters :+ pileupGroup)
+  def toClusters(pileupToNeighbors: Map[BreakpointPileup, PileupGroup]): Seq[PileupGroup] = {
```

Comment:
Could not add a test case that made the previous implementation overflow due to restrictions on sharing that data.  Not sure it's worth it, but happy to add a few more test cases if you think that would be worth it.